2024-11-19 19:02:46 INFO     Finished wandb init.
2024-11-19 19:02:49 INFO     Train dataset: Dataset({
    features: ['id', 'question', 'context', 'answers'],
    num_rows: 12294
})
2024-11-19 19:02:50 INFO     Prompt is: Answer the following question as briefly as possible.
Question: In April, which sportsman married his childhood sweetheart Kim Sears?
Answer: andy murray

Question: From the Latin for argentum, which element, with an atomic number of 47, uses the symbol Ag?
Answer: silver

Question: In which English city will you find the Ashmolean museum?
Answer: oxford

Question: In which city was John Lennon murdered?
Answer: new york

Question: What was Groucho Marx's real first name?
Answer: julius


Loading checkpoint shards: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2/2 [00:10<00:00,  5.46s/it]
2024-11-19 19:03:09 WARNING  Some parameters are on the meta device because they were offloaded to the disk.
2024-11-19 19:03:10 ERROR    Run failed with error: The model has been loaded with `accelerate` and therefore cannot be moved to a specific device. Please discard the `device` argument when creating your pipeline object.
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/Users/darius/Code/deep-learning/llms/mlo/llm-reasoning-uncertainty/evaluation/generate_answers.py", line 365, in <module>
    main(args)
  File "/Users/darius/Code/deep-learning/llms/mlo/llm-reasoning-uncertainty/evaluation/generate_answers.py", line 133, in main
    model = ModelWithProphetWrapper(
            ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/darius/Code/deep-learning/llms/mlo/llm-reasoning-uncertainty/techniques/speculative_decoding.py", line 307, in __init__
    self.pipeline = pipeline(
                    ^^^^^^^^^
  File "/Users/darius/miniconda3/envs/semantic_uncertainty/lib/python3.11/site-packages/transformers/pipelines/__init__.py", line 1164, in pipeline
    return pipeline_class(model=model, framework=framework, task=task, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/darius/miniconda3/envs/semantic_uncertainty/lib/python3.11/site-packages/transformers/pipelines/text_generation.py", line 96, in __init__
    super().__init__(*args, **kwargs)
  File "/Users/darius/miniconda3/envs/semantic_uncertainty/lib/python3.11/site-packages/transformers/pipelines/base.py", line 874, in __init__
    raise ValueError(
ValueError: The model has been loaded with `accelerate` and therefore cannot be moved to a specific device. Please discard the `device` argument when creating your pipeline object.
